framework/src/application/application.ts

```diff
@@ -694,11 +699,26 @@ export class Application {
 				registeredTransactions: this.getTransactions(),
 			},
 			logger: this.logger,
+			// TODO: Remove the storage with PR 5257
 			// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
 			storage: this.storage,
+			forgerDB: this._getDBInstance(nodeConfigs, 'forger.db'),
 			applicationState: this._applicationState,
 		});
 
 		return node;
 	}
+
+	private _getDBInstance(
+		options: Partial<ApplicationConfig>,
+		dbName: string,
+	): KVStore {
+		// eslint-disable-next-line @typescript-eslint/restrict-template-expressions
+		const dbPath = `${options.rootPath}/${options.label}/data/${dbName}`;
+		if (!fs.existsSync(dbPath)) {
```

Comment:
No need to create the `${options.rootPath}/${options.label}/data/` here, it is created somewhere else.
If the `........./data`  exists, actual `forger.db` will be created by the KVStore